shells/mrsh/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           sourcehut 1.0
+
+sourcehut.setup     emersion mrsh a858396b
```

Comment:
I know the situation with the project not having any releases is not ideal.
Unfortunately, I cannot tell when or if a release will be made. The only thing I could find is this GitHub issue https://github.com/emersion/mrsh/issues/8.

Reply:
A grep across the [macports-ports](https://github.com/macports/macports-ports) tree revealed multiple ports with a version "0":

* [cws2fws](https://github.com/macports/macports-ports/blob/master/multimedia/cws2fws/Portfile#L4)
* [libhsplasma](https://github.com/macports/macports-ports/blob/master/devel/libhsplasma/Portfile#L7)
* [litebrowser](https://github.com/macports/macports-ports/blob/master/www/litebrowser/Portfile#L14)
* [plovr](https://github.com/macports/macports-ports/blob/master/devel/plovr/Portfile#L9)
* [PothosLiquidDSP](https://github.com/macports/macports-ports/blob/master/science/PothosLiquidDSP/Portfile#L18)
* [rnnoise](https://github.com/macports/macports-ports/blob/master/audio/rnnoise/Portfile#L7)
* [sqlitedbrowser](https://github.com/macports/macports-ports/blob/master/databases/sqlitedbrowser/Portfile#L7)
* [sse2neon](https://github.com/macports/macports-ports/blob/master/devel/sse2neon/Portfile#L9)
* [wireguard-go](https://github.com/macports/macports-ports/blob/master/net/wireguard-go/Portfile#L6)

Maybe instead of version 0.0.0 this port could also have the commit date appended?
Or instead of 0.0.0 we use the commit date as the version and once they make a release bump the epoch?

Reply:
Implemented.